data/names.txt

```diff
@@ -3069,80 +3069,16 @@ phrase "bad outcomes"
 
 phrase "pirate"
 	word
-		"Caleuche"
-		"Lady Lovibond"
-		"Flying Dutchman"
-		"Princess Augusta"
-		"Eliza Battle"
-		"Iron Mountain"
-		"Eurydice"
-		"Valencia"
-		"Octavius"
-		"Red Jenny"
-		"Ourang Medan"
-		"Sea Bird"
-		"Resolute"
-		"Mary Celeste"
-		"Resolven"
-		"Zebrina"
-		"Deering"
-		"Bavchimo"
-		"Governor Parr"
-		"Joyita"
-		"Teignmouth Electron"
-		"Ocean Wave"
```

Comment:
The ghost ship names appear to have been taken from [this Wikipedia page](https://en.wikipedia.org/wiki/Ghost_ship#Historically_attested). The *Ocean Wave* was from 1975, which was a bit too modern for my tastes, so it and subsequent ships were not moved to the new "historical ghost ships" category.